package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "eslint-plugin-txo-typescript",
+  "version": "1.0.0",
+  "description": "Technology Studio - Eslint typescript plugin",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/technology-studio/eslint-plugin-txo-typescript.git"
+  },
+  "author": {
+    "name": "Technology Studio",
+    "email": "npm@technologystudio.sk",
+    "url": "https://github.com/technology-studio/"
+  },
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "lint": "eslint --format=node_modules/eslint-formatter-pretty --ext .jsx,.js,.ts,.tsx .",
+    "fixcode": "eslint --format=node_modules/eslint-formatter-pretty --ext .jsx,.js,.ts,.tsx . --fix",
+    "git-hook": "yarn -s lint",
+    "ncu-all": "ncu -i",
+    "compile": "tsc",
+    "prepare": "husky install && yarn compile",
+    "sanity": "yarn lint && tsc --noEmit && yarn outdated && echo 'success'",
+    "semantic-release": "semantic-release",
+    "test": "yarn compile && cp -r lib node_modules/eslint-plugin-txo-typescript/ && yarn lint"
```

Comment:
@rostislav-simonik please check if this is okay, at the time of `yarn install` the `lib` folder doesn't exist yet, so the CI check fails when running `yarn test`

Reply:
yarn install --dev file:. instead

Reply:
ok, we use copy due broken yarn install for local package because  it copies also node_modules